jac-uk/admin
# #1978 Move panel packs to reports

## What's included?
Close #1978 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
1. Go to Reports of an exercise and check if `Sift`, `Selection day`, and `Scenario Responses` display.
2. Go to `Sift`, `Selection day`, and `Scenario Responses` and check if the functionalities work.

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Screenshot:

![image](https://github.com/jac-uk/admin/assets/79906532/0890bbb2-bb3f-46d9-ba6f-5e9e63387696)

## Related permissions
Have permissions been considered for this functionality?
- No permission changes required

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
